components/google_ads/README.md

```diff
@@ -18,3 +18,5 @@ and campaigns programmatically. With the API, you can automate common tasks,
 such as creating and managing campaigns, adding and removing keywords, and
 adjusting bids. You can also use the API to get information about your
 campaigns, such as campaign stats, keyword stats, and ad performance.
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Reply:
Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).